common/amount.h

```diff
@@ -98,6 +98,13 @@ bool amount_msat_less_sat(struct amount_msat msat, struct amount_sat sat);
 /* Is msat <= sat? */
 bool amount_msat_less_eq_sat(struct amount_msat msat, struct amount_sat sat);
 
+/* Does value overflow u32 bit */
+bool amount_msat_overflow(struct amount_msat msat);
```

Comment:
This should probably have a `u32` in its name.

Reply:
addressed